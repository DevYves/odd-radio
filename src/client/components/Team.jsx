import React, { Component } from 'react';

class Team extends Component {
  render() {
    return (
      <div className="container masthead-info">
        <div className="row center">
          <h5>The Team</h5>
        </div>
        <div className="row">
          <p>The team behind oddrad.io:</p>
          <div className="row">
            <p>Johnnie Regalado</p>
            <p>Yves Frepon</p>
            <p>Colin Richardson</p>
            <p>Sebastian Bethell</p>
          </div>
          <p>oddrad.io was developed as a final project for Lighthouse Labs web development bootcamp.</p>
        </div>
      </div>
    )
  }
}

export default Team;