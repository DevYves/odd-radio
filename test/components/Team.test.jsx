import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

import Team from '../../src/client/components/Team';

describe('TeamComponent', () => {

  const subject = <Team />;

  it('matches snapshot', () => {
    expect(renderer.create(subject)).toMatchSnapshot();
  });
});