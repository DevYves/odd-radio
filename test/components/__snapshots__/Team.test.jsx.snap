// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamComponent matches snapshot 1`] = `
<div
  className="container masthead-info"
>
  <div
    className="row center"
  >
    <h5>
      The Team
    </h5>
  </div>
  <div
    className="row"
  >
    <p>
      The team behind oddrad.io:
    </p>
    <div
      className="row"
    >
      <a
        href="https://github.com/johnniereg"
        target="blank"
      >
        <p>
          Johnnie Regalado
        </p>
      </a>
      <a
        href="https://github.com/DevYves"
        target="blank"
      >
        <p>
          Yves Frepon
        </p>
      </a>
      <a
        href="https://github.com/RyukyuColin"
        target="blank"
      >
        <p>
          Colin Richardson
        </p>
      </a>
      <a
        href="https://github.com/SebastianBethell"
        target="blank"
      >
        <p>
          Sebastian Bethell
        </p>
      </a>
    </div>
    <p>
      oddrad.io was developed by brand new programmers as a final project for the Lighthouse Labs web development bootcamp.
    </p>
  </div>
</div>
`;
